src/core/components/AMInstallButton/index.js

```diff
@@ -119,14 +120,16 @@ export class AMInstallButtonBase extends React.Component<InternalProps> {
   };
 
   installExtension = async (event: SyntheticEvent<HTMLAnchorElement>) => {
-    const { addon, enable, install } = this.props;
+    const { addon, enable, getCurrentStatus, install } = this.props;
 
     event.preventDefault();
     event.stopPropagation();
 
     await install();
 
-    if (addon.type === ADDON_TYPE_STATIC_THEME) {
+    const isEnabled = await getCurrentStatus();
+
+    if (addon.type === ADDON_TYPE_STATIC_THEME && !isEnabled) {
```

Comment:
I think we should call `getCurrentStatus()` inside the `if`, because it is only needed for static themes.

Reply:
Given that the status is likely up-to-date, can we maybe just do:

```
if (addon.type === ADDON_TYPE_STATIC_THEME && status !== ENABLED) {
  ...
}
```